resources/views/docs/desktop/1/the-basics/notifications.md

```diff
@@ -40,8 +40,97 @@ Notification::title('Hello from NativePHP')
     ->show();
 ```
 
+### Notification Reference
+
+To keep track of different notifications, each notification gets a reference once created. You can manually set a reference using the `reference()` method. 
+By default, a unique ID is generated as the reference. Once the notification is shown, the reference is stored in the notification class.
+
+```
+$notification = Notification::title('Hello from NativePHP')->show();
+$notification->reference; // <-- This property contains the reference
+```
```

Comment:
This section feels a bit like duplication of the portion below... I think what we need is an example of how to use the reference inside an event handler